crates/bevy_scene/src/serde.rs

```diff
@@ -642,7 +642,7 @@ mod tests {
 
         let deserialized_scene = scene_deserializer.deserialize(&mut deserializer).unwrap();
 
-        let mut map = EntityMap::default();
+        let mut map = Default::default();
```

Comment:
Let's write type for clarity and consistency with the rest of the engine here and in other places:
```suggestion
        let mut map = HashMap::default();
```

Reply:
Fair enough. Only used it to avoid the couple of lines associated with using the HashMap.